addons/test_event_full/tests/test_performance.py

```diff
@@ -292,7 +292,7 @@ def test_registration_create_form_customer(self):
         """ Test a single registration creation using Form """
         event = self.env['event.event'].browse(self.test_event.ids)
 
-        with freeze_time(self.reference_now), self.assertQueryCount(event_user=229):  # tef only: 209 - com runbot 215
+        with freeze_time(self.reference_now), self.assertQueryCount(event_user=330):  # tef only: 210 - com runbot 216
```

Comment:
Wait, shouldn't it be 230 ?

Reply:
Ho yeah, nice catch ! I am making the change.